src/others/vg_lite_tvg/vg_lite_tvg.cpp

```diff
@@ -711,6 +713,67 @@ extern "C" {
         return VG_LITE_SUCCESS;
     }
 
+    vg_lite_error_t vg_lite_set_stroke(vg_lite_path_t * path,
+                                       vg_lite_cap_style_t cap_style,
+                                       vg_lite_join_style_t join_style,
+                                       vg_lite_float_t line_width,
+                                       vg_lite_float_t miter_limit,
+                                       vg_lite_float_t * dash_pattern,
+                                       vg_lite_uint32_t pattern_count,
+                                       vg_lite_float_t dash_phase,
+                                       vg_lite_color_t color)
+    {
+        if(!path || line_width <= 0) {
+            return VG_LITE_INVALID_ARGUMENT;
+        }
+
+        if(miter_limit < 1.0f) {
+            miter_limit = 1.0f;
+        }
+
+        if(!path->stroke) {
+            path->stroke = (vg_lite_stroke_t *)malloc(sizeof(vg_lite_stroke_t));
+
+            if(!path->stroke) {
+                return VG_LITE_OUT_OF_RESOURCES;
+            }
+
+            memset(path->stroke, 0, sizeof(vg_lite_stroke_t));
```

Comment:
Please use `lv_memset`
(maybe in a specific platform the implementation is faster / more optimized than the default `memset` )